packages/ui/src/components/ImageCarousel/ImageCarousel.tsx

```diff
@@ -0,0 +1,42 @@
+'use client'
+import { ImageCard, ImageCardData, ImageCardDataType } from '@repo/ui'
+import { Swiper, SwiperSlide } from 'swiper/react'
+import 'swiper/swiper-bundle.css'
+
+type Props = {
+  ImageCardData: ImageCardDataType[]
+  title?: string
+}
+export const ImageCarousel: React.FC<Props> = ({
+  title = 'External Tools',
+}) => {
+  return (
+    <>
+      <div className="flex flex-row items-center justify-between px-8">
+        <p className="font-semibold text-lg py-8 leading-5">{title}</p>
+        {/* <a
```

Comment:
Make it work. updating the grid styles to this in the layout file helped.

grid grid-cols-1 md:grid-cols-[320px_calc(100%-320px)]

and display the anchor tag as well.

Reply:
Just reviewed Shiyamal's PR o profile cards. Its content other than the sidebar is occupying the entire remaining space without any change. Not sure what exactly is the reason its not working in your PR.